examples/advanced/random_forest/jobs/random_forest_base/app/config/config_fed_server.json

```diff
@@ -42,6 +42,7 @@
         "shareable_generator_id": "shareable_generator",
         "train_task_name": "train",
         "train_timeout": 0,
+        "allow_empty_global_weights": 1,
```

Comment:
Use "1" here might be confusing to the users.
Please change to `"allow_empty_global_weights": true,`